src/main/java/com/ibm/dfdl/sample/sax/reader/DFDLReader1.java

```diff
@@ -0,0 +1,262 @@
+/**********************************************************************
+* Licensed Materials - Property of IBM
+*
+* (C) Copyright IBM Corp. 2014
+*
+* US Government Users Restricted Rights - Use, duplication, or
+* disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+**********************************************************************/
```

Comment:
The licensing isn't clear to me for some of these files. Are we allowed to redistribute these files from IBM?

Reply:
We are not. I'll have to pull this all down.

Reply:
Actually, my plan is to write a replacement of this class in scala.